packages/dashboard/src/routes/index.svelte

```diff
@@ -1,2 +1,7 @@
 <script lang="ts">
+	import CreateOrganization from '$lib/components/CreateOrganization.svelte';
+	import CreateProject from '$lib/components/CreateProject.svelte';
 </script>
+
+<!-- <CreateOrganization /> -->
```

Comment:
You were on the right track using component props. But instead of creating two components (createProjec and createOrganization), the CreateEntity component is enough. 

You can change this code to 
```Svelte
<CreateEntity title="Your projects" description="" name="project" />
<CreateEntity title="Your organizations" description="" name="organization" />
```

Now we only have to modify `CreateEntity` for changes instead of three components namely `CreateEntity, CreateProject, CreateOrganization`.